team2_pii.py

```diff
@@ -14,4 +14,16 @@ def find_us_ssn(text):
     match = re.search(r'([0-9]{2}[1-9]|[0-9][1-9][0-9]|[1-9][0-9]{2})[-\s]([0-9][1-9]|[1-9][0-9])[-\s]([0-9]{3}[1-9]|[0-9]{2}[1-9][0-9]|[0-9][1-9][0-9]{2}|[1-9][0-9]{3})', text)
     if match:
         return True
-    return False
\ No newline at end of file
+    return False
+
+def find_us_twitter_handle(text):
+    match = re.search(r'^[@](\w){1,15}$', text)
```

Comment:
Keep in mind this won't match something like
```
Twitter ID: @John_Smith
```
This may or may not present an issue for you in sprint-4 depending on your overall design.  No need to fix for this sprint, just pointing it out.

Same comment for your email handling